docs/read-side.md

```diff
@@ -272,7 +271,7 @@ We recommend that you store Read Model data in a denormalized form so that your
 
 A projection function is used to accumulate the event data in a **Read Model storage**. Each projection function takes the storage object and event information. The event information includes the aggregateID, timestamp and payload.
 
-You can communicate with the store using the standard API. The code sample below demonstrates a Read Model projection function implementation:
+You can communicate with the store using the [standard API](api-reference#read-model-store-interface). The code sample below demonstrates a Read Model projection function implementation:
```

Comment:
Do not use 'using'

Reply:
Changed to 'You can use the [standard API](api-reference#read-model-store-interface) to communicate with store.'